Sources/XcodeGenKit/SchemeGenerator.swift

```diff
@@ -377,3 +377,10 @@ extension PBXProductType {
         }
     }
 }
+
+extension Target {
+    var shouldExecuteOnLaunch: Bool {
```

Comment:
What do you think about moving this into https://github.com/yonaskolb/XcodeGen/blob/master/Sources/ProjectSpec/XCProjExtensions.swift where all the other similar helpers are?